### DISCUSSION TITLE
TumblThree failes to download any posts from some of Tumblr Tagged searches

### DISCUSSION THREAD

**Discussion by Klimax** [TumblThreeApp/TumblThree#307] [Q&A (Question & Answer)]
**Describe the bug**
When attempting to download some of Tumblr Tagged searches, app fails with error/exception: "A post from jadejohn was not parsable." or no error at all (case of animegirl).
Example of such tagged search: https://www.tumblr.com/tagged/animegirl or https://www.tumblr.com/tagged/jadejohn

Setting "posts per page" has no effect.

**To Reproduce**
1. Add https://www.tumblr.com/tagged/jadejohn as new blog
2. Start crawler
3. Error/exception "A post from jadejohn was not parsable." appears and no crawl and no files are downloaded.

**Expected behavior**
Download of files like with other tag searches.

**Desktop (please complete the following information):**
 - TumblThree version: 2.5.6.292
 - OS: Windows 10 Pro x64

**Comment by thomas694** [comment]
Thanks for reporting a bug. We'll look at it.

**Comment by thomas694** [comment]
Tumblr has changed its behavior. TumblThree cannot find anymore the content and this error is shown.

Watch it in your browser.
If you open https://www.tumblr.com/tagged/animegirl, it redirects you to https://www.tumblr.com/search/animegirl.
If you open https://www.tumblr.com/tagged/jadejohn, it loads https://www.tumblr.com/tagged/jadejohn?sort=top.

For some unknown reason Tumblr doesn't like to start a tag search for some tags anymore. That's visible, if you directly use the url or click on a post's tag. But if you enter the tag word with hash character in the search box and press on the tag suggestion in the popup, it redirects to https://www.tumblr.com/tagged/%23animegirl. Weird behavior.

Maybe TumblThree needs to be adjusted to detect that situation and retry it with the alternative url. We'll see.

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
